src/code/z_demo.c

```diff
@@ -95,19 +95,17 @@ void func_80064558(GlobalContext* globalCtx, CutsceneContext* csCtx) {
     }
 }
 
-#ifdef NON_MATCHING
-// minor ordering and regalloc differences
 void func_800645A0(GlobalContext* globalCtx, CutsceneContext* csCtx) {
-    CutsceneStateHandler handler;
+    Input *player1 = &globalCtx->state.input[0];
```

Comment:
I'm not sure about using the term "player" in this context.
Also pointers should have the star on the left side based on our style.

Reply:
renamed to "pad1" and corrected the star.

Reply:
pad1 makes sense but also might be confused for stack padding (lmao). Maybe controller1?

Reply:
I can do that, I mostly named it that because `pad1->padPressed`